index.bs

```diff
@@ -890,7 +890,9 @@ When the {{MLContext/[[contextType]]}} is set to [=context type/default=] with t
   </summary>
     1. Let |transferredViews| be a new {{MLNamedArrayBufferViews}}.
     1. [=map/For each=] |name| → |view| of |views|:
```

Comment:
I think part of the core of #351 is that if this loop aborts early then the state of the buffers in `views` is unclear since some of them might've been transferred. If we can guarantee that  none of these other steps can fail then we could run a loop before this to check that all the buffers are detachable and then not try transferring any of them so that we can assert that an error is not thrown.

Reply:
Ah, thanks.